### ISSUE TITLE
df -k fails to select 1024-byte blocks

### ISSUE THREAD

**Issue by jgarzik** [rustcoreutils/posixutils-rs#301]
After recent changes (it worked a while ago), `df -k` fails to select 1024-byte blocks.

cc @fox0 

```
% ./target/release/df | head -2
Filesystem     512-blocks       Used  Available Capacity Mounted on
/dev/disk1s4s1  976490576  152157960  824332616      16% /         

% ./target/release/df -k | head -2
Filesystem     512-blocks       Used  Available Capacity Mounted on
/dev/disk1s4s1  976490576  152157952  824332624      16% /    
```

**Comment by fox0** [comment]
https://github.com/rustcoreutils/posixutils-rs/pull/302

**Comment by jgarzik** [comment]
Fixed by #302
